_posts/2023-05-05-NASA-grant-announcement.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "One small step for NASA, one big step for open research"
+subtitle: "OLS and MetaDocencia to deliver OpenCore Science Training"
```

Comment:
![image](https://github.com/open-life-science/open-life-science.github.io/assets/1842467/14f81275-f096-4904-9857-e9e0454ba48c)

The subtitle is not displayed on the top. Maybe we can use `description` instead (as in the `index.md`)

Reply:
Yes, I think that might be should...

Reply:
Yes, I think that might be better...